sdks/java/core/src/main/java/org/apache/beam/sdk/util/TriggerContextFactory.java

```diff
@@ -53,16 +53,13 @@
 
   private final WindowFn<?, W> windowFn;
   private StateInternals<?> stateInternals;
-  // Future triggers may be able to exploit the active window to state address window mapping.
-  @SuppressWarnings("unused")
-  private ActiveWindowSet<W> activeWindows;
   private final Coder<W> windowCoder;
 
   public TriggerContextFactory(WindowFn<?, W> windowFn,
       StateInternals<?> stateInternals, ActiveWindowSet<W> activeWindows) {
+    // Future triggers may be able to exploit the active window to state address window mapping.
```

Comment:
We're still passing in `activeWindows` to the constructor, so the implementation could take advantage of this in the future. The comment explains why we have an unused constructor param, so I prefer to leave it.